src/System.Reflection.Metadata/src/System/Reflection/Metadata/TypeSystem/TypeDefinition.cs

```diff
@@ -53,7 +53,7 @@ public TypeAttributes Attributes
         }
 
         /// <summary>
-        /// Is the type is nested.
+        /// Indicates whether this is the nested type.
```

Comment:
This should read "Indicates whether this is *a* nested type" and not "... *the* nested type.